letsdrill.rb

```diff
@@ -1,13 +1,21 @@
 def get_letter_grade(integer)
  
   #Put your code here!
-
+  case integer
+    when  90..100 then "A"
+    when  80..89 then "B"
+    when  70..79 then "C"
+    when  60..69 then "D"
+    else "F"
+  end
 end
 
 def shortest_string(array)
 
   #Put your code here!
-
+  shortest = array
+  shortest = shortest.sort_by(&:length)
```

Comment:
What prevents you from making this a oneliner? It's still quite readable as `array.sort_by(&:length).first`